src/main/java/frc/robot/commands/DepositCommand.java

```diff
@@ -0,0 +1,125 @@
+package frc.robot.commands;
+
+import com.kauailabs.navx.frc.AHRS;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.commands.swerve.FollowPathCommand;
+import frc.robot.subsystems.drivetrain.BaseDrivetrain;
+import frc.robot.subsystems.drivetrain.BaseSwerveSubsystem;
+
+public class DepositCommand extends CommandBase {
+    private final BaseSwerveSubsystem driveSubsystem; // changed to BASE SWERVE SUBSYS to allow for getRobotPosition() 
+    private final AHRS ahrs; 
+
+    private final boolean red;
```

Comment:
I think this doesn't matter if you pass in positions to the constructor. If you're on the red team, you just pass in a red position.